tripper/triplestore.py

```diff
@@ -686,14 +730,13 @@ def _add_function_fno(self, func, expects, returns, base_iri):
         self.add((func_iri, RDF.type, FNO.Function))
         self.add((func_iri, FNO.expects, parlist))
         self.add((func_iri, FNO.returns, outlist))
-        if doc:
-            self.add((func_iri, DCTERMS.description, en(doc)))
+        if inspect.getdoc(func):
```

Comment:
This is slightly inefficient, since it calls `inspect.getdoc(func)` twice...

Reply:
I'm betting on clever Python caching here. But you are correct.
My goal here was to minimize the number of local variables.